src/appdistribution/client.ts

```diff
@@ -92,7 +92,7 @@ export class AppDistributionClient {
       method: "POST",
       path: `/upload/v1/${appName}/releases:upload`,
       headers: {
-        "X-Goog-Upload-File-Name": distribution.getFileName(),
+        "X-Goog-Upload-File-Name": encodeURIComponent(distribution.getFileName()),
```

Comment:
Does this header use URI encoding (i.e. how is it decoded on the receiving end)?

Reply:
Yep, we already decode this on the server-side. This is actually the same technique that the Firebase console uses today.